Client/game_sa/CHandlingEntrySA.h

```diff
@@ -96,78 +96,78 @@ class CHandlingEntrySA : public CHandlingEntry
-    float GetSuspensionDamping() const { return m_Handling.fSuspensionDamping; }
-    float GetSuspensionHighSpeedDamping() const { return m_Handling.fSuspensionHighSpdDamping; }
-    float GetSuspensionUpperLimit() const { return m_Handling.fSuspensionUpperLimit; }
-    float GetSuspensionLowerLimit() const { return m_Handling.fSuspensionLowerLimit; }
-    float GetSuspensionFrontRearBias() const { return m_Handling.fSuspensionFrontRearBias; }
-    float GetSuspensionAntiDiveMultiplier() const { return m_Handling.fSuspensionAntiDiveMultiplier; }
+    float GetSuspensionForceLevel() const noexcept { return m_Handling.fSuspensionForceLevel; }
+    float GetSuspensionDamping() const noexcept { return m_Handling.fSuspensionDamping; }
+    float GetSuspensionHighSpeedDamping() const noexcept { return m_Handling.fSuspensionHighSpdDamping; }
+    float GetSuspensionUpperLimit() const noexcept { return m_Handling.fSuspensionUpperLimit; }
+    float GetSuspensionLowerLimit() const noexcept { return m_Handling.fSuspensionLowerLimit; }
+    float GetSuspensionFrontRearBias() const noexcept { return m_Handling.fSuspensionFrontRearBias; }
+    float GetSuspensionAntiDiveMultiplier() const noexcept { return m_Handling.fSuspensionAntiDiveMultiplier; }
 
-    float GetCollisionDamageMultiplier() const { return m_Handling.fCollisionDamageMultiplier; }
+    float GetCollisionDamageMultiplier() const noexcept { return m_Handling.fCollisionDamageMultiplier; }
 
-    unsigned int GetHandlingFlags() const { return m_Handling.uiHandlingFlags; }
-    unsigned int GetModelFlags() const { return m_Handling.uiModelFlags; }
-    float        GetSeatOffsetDistance() const { return m_Handling.fSeatOffsetDistance; }
-    unsigned int GetMonetary() const { return m_Handling.uiMonetary; }
+    unsigned int GetHandlingFlags() const noexcept { return m_Handling.uiHandlingFlags; }
+    unsigned int GetModelFlags() const noexcept { return m_Handling.uiModelFlags; }
+    float        GetSeatOffsetDistance() const noexcept { return m_Handling.fSeatOffsetDistance; }
+    unsigned int GetMonetary() const noexcept { return m_Handling.uiMonetary; }
 
-    eLightType    GetHeadLight() const { return static_cast<eLightType>(m_Handling.ucHeadLight); }
-    eLightType    GetTailLight() const { return static_cast<eLightType>(m_Handling.ucTailLight); }
-    unsigned char GetAnimGroup() const { return m_Handling.ucAnimGroup; }
+    eLightType    GetHeadLight() const noexcept { return static_cast<eLightType>(m_Handling.ucHeadLight); }
+    eLightType    GetTailLight() const noexcept { return static_cast<eLightType>(m_Handling.ucTailLight); }
+    unsigned char GetAnimGroup() const noexcept { return m_Handling.ucAnimGroup; }
 
-    std::uint16_t GetVehicleID() const { return static_cast<std::uint16_t>(m_Handling.iVehicleID); }
+    eHandlingTypes GetHandlingID() const noexcept { return static_cast<eHandlingTypes>(m_Handling.iVehicleID); }
```

Comment:
Why did you rename it?

Reply:
More correctly, because it refers to HT_ and not to anything else

Reply:
HT?

Reply:
Simply put, these indicators are related to eHandlingTypes, so I thought it was right to change the name

Reply:
It will be confusing with existing GetHandlingID (unless those references are just you renaming GetVehicleID occurences with GetHandlingID)

Reply:
Okay, I'll change the name.